pkg/kudoctl/util/kudo/kudo.go

```diff
@@ -127,13 +129,30 @@ func (c *Client) InstanceExistsInCluster(operatorName, namespace, version, insta
 	return true, nil
 }
 
+// Populate the GVK from scheme, since it is cleared by design on typed objects.
+// https://github.com/kubernetes/client-go/issues/413
+func setFirstGVK(object runtime.Object) error {
```

Comment:
mentally parsing the func name "FirstGVK"...  I don't understand... is this initializing?

Reply:
We're setting GVK. The value we use is the **first** of the available values in `Scheme.ObjectKinds`.
Suggestions for a better name most welcome.